modules/shared/src/test/scala/org/tessellation/schema/TransactionSuite.scala

```diff
@@ -59,7 +59,7 @@ object TransactionSuite extends ResourceSuite with Checkers {
   }
 
   test("Hash for a new Transaction schema should be the same as hash for old Transaction schema") { implicit kryo =>
-    val expectedHash = "1017a072225263aa502d9dcd22a04455f9b063cc19166356be50b4730afc44f6"
+    val expectedHash = Hash("1017a072225263aa502d9dcd22a04455f9b063cc19166356be50b4730afc44f6")
```

Comment:
Sounds good :+1:

Reply:
Does it make sense to change all at once in separate PR? I'd keep that one strictly for Scala update

Reply:
I guess `Hash` being a newtype wrapping a string value, compared to another string value could be a reason here as well. I think it could be underlying value of a newtype that was used for generating the Eq (so Eq for strings).

Reply:
@marcinwadon I think we can try that. It shouldn't be too painful I hope.